Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'puma', '~> 3.0'
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 gem 'rack-cors'
 
-gem 'ontohub-models', github: 'ontohub/ontohub-models', branch: 'master', require: false
+gem 'ontohub-models', github: 'ontohub/ontohub-models', branch: 'create_models_as_rails_engine'
```

Comment:
It should, but only after the master branch is up to date. I need to change this line after merging ontohub-models/pulls/1 and before merging this one. Or maybe it's better to specify a commit?